wolfSSL/src/chapter02.md

```diff
@@ -1198,6 +2121,110 @@ Can be defined when building for FreeRTOS (<https://www.freertos.org>). If using
 
 Can be defined when building for the FreeRTOS windows simulator (<https://www.freertos.org>).
 
+#### WOLFSSL_CHIBIOS
+
+Can be defined when building for ChibiOS RTOS.
+
+#### WOLFSSL_CMSIS_RTOS
+
+Can be defined when building for Mbed CMIS-RTOS.
+
+#### WOLFSSL_CMSIS_RTOSv2
+
+Can be defined when building for Mbed CMIS-RTOSv2.
+
+#### WOLFSSL_LWIP_NATIVE
+
+Use in platforms for LWIP native.
+
+#### WOLFSSL_DEOS
+
+You can enable the wolfSSL support for Deos RTOS available [here](https://www.ddci.com/products_deos_do_178c_arinc_653/) using this define.
+
+#### WOLFSSL_ESPIDF
+
+Can be defined when building for ESP-IDF.
+
+#### WOLFSSL_LINUXKM
+
+Use if building for Linux Kernel Module.
+
+#### WORD64_AVAILABLE
+
+Platform with 64-bit CPU registers.
```

Comment:
Portability macro to indicate 64-bit types are supported. Typically its better to use `#define SIZEOF_LONG_LONG 8`.